src/routes/scope-selection/ScopeSelectionScreen.tsx

```diff
@@ -92,9 +128,37 @@ const ScopeSelectionScreen = ({ navigation, route }: IntroScreenProps): JSX.Elem
           )}
           <StyledText>{getLabels().scopeSelection.selectProfession}</StyledText>
         </TextContainer>
-        <ServerResponseHandler error={error} loading={loading} refresh={refresh}>
-          <DisciplineContainer>{disciplineItems}</DisciplineContainer>
-        </ServerResponseHandler>
+        <SearchContainer>
+          <SearchBar
+            query={searchString}
+            setQuery={input => {
+              setSearchString(input)
+              setShowSearchResults(input.length > 0)
+            }}
+            placeholder={getLabels().scopeSelection.searchProfession}
+          />
+        </SearchContainer>
+        {showSearchResults ? (
+          <ScopeContainer>
+            {filteredProfessions?.map(profession => (
+              <Pressable
+                key={profession.id}
+                onPress={async () => {
+                  await pushSelectedProfession(profession.id).then(() => {
```

Comment:
Actually, I think the buttons should not be pressable, once you've selected a profession. Maybe we can show a loading screen or just disable the buttons.

Reply:
Hmm, but if someone wants to select a second profession, I don't see a problem with that. You already can't select the same profession twice anymore since it doesn't show up in the search anymore.

Reply:
But when I search for the profession it is still visible. At least if I add the module via the settings
<img width="376" alt="Bildschirmfoto 2024-01-17 um 15 21 34" src="https://github.com/digitalfabrik/lunes-app/assets/28013368/4c4cd06c-a1ca-457c-b6c4-b8a26fe2dd12">